jmix-reports/reports/src/main/java/io/jmix/reports/yarg/formatters/impl/XlsxFormatter.java

```diff
@@ -918,6 +1010,32 @@ protected List<Cell> copyCells(Range templateRange, BandData bandData, Row newRo
         return resultCells;
     }
 
+    private void copyTemplateImages(WorksheetPart worksheetPart, Range templateRange, Cell templateCell, Row newRow, Cell newCell) {
+        int newRowNum = newRow.getR().intValue() - 1;
+        int newColNum = computeColumnIndex(newCell.getR()) - 1;
+
+        List<XlsxImage> images = templateImages.get(templateRange.getSheet() + "_" + templateCell.getR());
+
+        if (images != null && images.size() > 0) {
+            for (XlsxImage image : images) {
+                try {
+                    org.docx4j.openpackaging.parts.DrawingML.Drawing drawing = getOrCreateWorksheetDrawing(result.getPackage(), worksheetPart);
+                    org.docx4j.openpackaging.parts.DrawingML.Drawing exxdrawing = getOrCreateWorksheetDrawing(result.getPackage(), worksheetPart);
+                    BinaryPartAbstractImage imagePart = BinaryPartAbstractImage.createImagePart(result.getPackage(), drawing,
+                            image.getPictureData());
+                    String imageRelID = imagePart.getSourceRelationships().get(0).getId();
+                    RelationshipsPart relPart = attachImageToCell(drawing,
+                            newColNum, newRowNum, image, imageRelID);
+                    result.getPackage().getParts().put(relPart);
+                } catch (InvalidFormatException e) {
+                    throw new RuntimeException(e);
+                } catch (Exception e) {
+                    throw new RuntimeException(e);
+                }
```

Comment:
Why two catch blocks? Add a short message to a RuntimeException.

Reply:
fixed